client/src/components/Mention/MentionHeading/MentionHeading.js

```diff
@@ -5,6 +5,7 @@ import { makeStyles } from "@material-ui/core/styles";
 const useStyles = makeStyles({
   root: {
     marginBottom: 5,
+    maxWidth: "95%",
```

Comment:
Why is maxwidth needed to be set to 95%?

Reply:
I tried it that way, but because the mood icon is set with absolute positioning, if the title is long, it moves under the mood icon. After I tested out a few widths, I found that 95% is enough for the title to steer clear of the mood icon without making it too small.